databricks/koalas/tests/test_indexops_spark.py

```diff
@@ -50,3 +50,16 @@ def test_multiindex_transform_negative(self):
             )
             s = ks.Series([45, 200, 1.2, 30, 250, 1.5, 320, 1, 0.3], index=midx)
             s.index.spark.transform(lambda scol: scol)
+
+    def test_series_apply_negative(self):
+        with self.assertRaisesRegex(
+            ValueError, "The output of the function.* pyspark.sql.Column.*int"
+        ):
+            self.kser.spark.apply(lambda scol: 1)
+
+        with self.assertRaisesRegex(AnalysisException, "cannot resolve.*non-existent.*"):
+            self.kser.spark.transform(lambda scol: F.col("non-existent"))
+
+    def test_index_apply_negative(self):
```

Comment:
Just curious, what is the reason we don't have positive tests for now ?
(Maybe because pandas doesn't have these kind of functions ?)

Reply:
Because I have doctests and these look handling the cases enough :-).

Reply:
@HyukjinKwon I got it! Makes sense, thanks :)